### PR TITLE
Added support for .coffee files

### PR DESCRIPTION
**Pull request by jhessin**
Only works with parenthesis for now.

### CODE REVIEW

**Review by Haroenv — APPROVED** [review]

File: lib/suggestions-provider.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default class {
   constructor(packagesIndex) {
     this.packagesIndex = packagesIndex;
     // This will work on JavaScript and CoffeeScript files, but not in js comments.
-    this.selector = '.source.js, .source.ts, .source.tsx';
+    this.selector = '.source.js, .source.ts, .source.tsx, .source.cofee';
```
Comment (rayrutjes) [inline lib/suggestions-provider.js]: Should be `.source.coffee`. Missing an "f".

**Review by rayrutjes — CHANGES_REQUESTED** [review]

File: lib/suggestions-provider.js (hunk 1 above)
Comment (Haroenv) [reply lib/suggestions-provider.js]: good catch, I didn't see the typo

Comment (jhessin) [reply lib/suggestions-provider.js]: I always do that (I'm spoiled by autocomplete). I changed it in the github editor otherwise I would have caught it.

**Comment by rayrutjes** [comment]
Merged in #29.

Released under 0.3.0.

Thanks for the contribution @jhessin ❤️

**Pull request closed without merging**